update issue templates (#10)
deislabs/osiris: .github/ISSUE_TEMPLATE/bug-report.md

---
name: Bug Report
about: Report a bug encountered while operating Osiris

---

> <!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!-->

Reviewer: ```suggestion
<!-- Please use this template while reporting a bug and provide as much info as possible. This would help us address your bug in a timely manner. Thanks!-->
```

Reviewer: That's my opinion. I'd still leave it to you to decide @krancour

**Environment:**
- Kubernetes distribution (e.g. AKS, ACS Engine, GKE, EKS, etc.):
- Kubernetes version (use `kubectl version`):
- Osiris version (specify by SHA or semver):
- Install method (specify exact `helm install` command used):

**What happened?**

**What you expected to happen?**

**How to reproduce it (as minimally and precisely as possible):**

> **Anything else we need to know?**

Reviewer: ```suggestion
**Anything else that we need to know?**
```
